modules/nlu/src/backend/engine2.ts

```diff
@@ -95,20 +102,18 @@ export default class Engine2 {
   }
 
   private async _makePredictors(model: Model, tools: Tools): Promise<Predictors> {
-    const { input, output, artefacts } = model.data
-    const processedIntents = output
-      ? output.intents
-      : await ProcessIntents(input.intents, model.languageCode, artefacts.list_entities, tools)
+    const { artefacts } = model.data
 
     const ctx_classifer = new tools.mlToolkit.SVM.Predictor(artefacts.ctx_model)
     const intent_classifier_per_ctx = _.toPairs(artefacts.intent_model_by_ctx).reduce(
       (c, [ctx, intentModel]) => ({ ...c, [ctx]: new tools.mlToolkit.SVM.Predictor(intentModel as string) }),
       {} as _.Dictionary<sdk.MLToolkit.SVM.Predictor>
     )
-    const slot_tagger = new CRFExtractor2(tools.mlToolkit)
-    slot_tagger.load(processedIntents, artefacts.slots_model)
+    const slot_tagger = new CRFExtractor2(tools.mlToolkit) // TODO change this pour MLToolkit.CRF.Tagger
```

Comment:
I don't like MLToolkit being part of the SDK, there's too much overhead.  What do you think to move it to `import {CRFTagger} from 'botpress/ml'` in the future?

Reply:
Agreed, will add this in the TODOs